source/_integrations/tesla_fleet.markdown

```diff
@@ -40,30 +40,135 @@ The Tesla Fleet API {% term integration %} exposes various sensors from Tesla ve
 
 ## Prerequisites
 
-You must have a [Tesla](https://tesla.com) account, and a [Developer Application](https://developer.tesla.com/en_AU/dashboard).
+You must have:
 
-### Developer Application
-
-You must [create your own application](https://developer.tesla.com/docs/fleet-api/getting-started/what-is-fleet-api#step-2-create-an-application) for the Tesla Fleet API and configure it as an [application credential](https://my.home-assistant.io/redirect/application_credentials).
-When creating the application, you must set the redirect URL to `https://my.home-assistant.io/redirect/oauth`, but the other URLs can be set as desired. You must also complete both [step 3](https://developer.tesla.com/docs/fleet-api/getting-started/what-is-fleet-api#step-3-generate-a-public-private-key-pair) and [step 4](https://developer.tesla.com/docs/fleet-api/getting-started/what-is-fleet-api#step-4-call-the-register-endpoint) before the application will be able to make API calls.
+- A [Tesla](https://tesla.com) account
+- A [Developer Application](https://developer.tesla.com/en_US/dashboard)
+- A web domain that you own and can host your public key file directly onto without needing to rely on URL redirects. Simple file-hosting services such as AWS S3 may help with this for free/cheap.
```

Comment:
I dont love mentioning _just_ AWS S3, we should either mention none or give a few different options. Core team might have specific ideas about this.

```suggestion
- A web domain and host that you can serve your public key file from. Some free options include:
  - [FleetKey.cc](https://fleetkey.cc)
  - [AWS S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteHosting.html)
  - [Cloudflare Pages](https://pages.cloudflare.com/)
  - [Firebase Hosting](https://firebase.google.com/docs/hosting)
  
```

Reply:
That makes sense. I saw on the other thread that you built this- thanks for that! Applying your suggestion.